src/Event/Productions/ProductionCommandHandler.php

```diff
@@ -135,4 +136,17 @@ private function assertEventExists(string $eventId)
             throw EventCannotBeAddedToProduction::becauseItDoesNotExist($eventId);
         }
     }
+
+    private function assertEventCanBeRemovedFromProduction(string $eventId, ProductionId $productionId)
+    {
+        try {
+            $this->eventRepository->get($eventId);
+            $production = $this->productionRepository->find($productionId);
+            if (!$production->containsEvent($eventId)) {
+                throw EventCannotBeRemovedFromProduction::becauseItDoesNotBelongToIt($eventId, $productionId);
```

Comment:
I wonder if this should throw. If the event does not belong to the production it could also be treated as a `no operation command`. To explain this with an example: the command to delete an event that is already deleted is also a `no operation command` and should not throw.

Reply:
So we could leave in the check if the event exists but make the other a no-op? Will fix.

Reply:
But when reading the ticket I see that is what is requested by @corneelwille